src/lib/comment-popover/comment-popover.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="comment-popover-content">
+  <div class="comment-header">
+    <h5>Add Note</h5>
+
+    <button
+      type="button"
+      class="btn close-comment close-svg"
+      tabindex="-1"
+      (click)="closeComment()"
+      (keyup)="closeComment()"
+    >
+      <fa-icon class="mx-auto my-auto" [icon]="I.faTimes" />
+    </button>
+  </div>
+  <div class="comment-border"></div>
+  <textarea
+    #commentInput
+    [(ngModel)]="comment"
+    rows="4"
+    placeholder="Enter your note"
+    style="height: 144px"
+    class="comment-textarea"
+    (keydown)="handleKeydown($event)"
```

Comment:
You can use `(keydown.enter)` and `(keydown.escape)` to call the event functions directly. Also helps with readability. Returning false will also stop the propagation if you want to limit it to just those keys.

Reply:
But we are using a combination of keys here as Mike suggested, so I guess we can't use this method for that.